polly/htmx_endpoints.py

```
@@ -570,30 +602,30 @@ async def open_poll_now_htmx(
-                db.commit()
+                await _revert_poll_status_to_scheduled(
+                    poll_id, open_time=prev_open_time
+                )
                 return templates.TemplateResponse(
                     "htmx/components/inline_error.html",
                     {"request": request, "message": result.get("error", "Error opening poll")},
                 )
-            
+
             logger.info(f"Poll {poll_id} opened immediately by user {current_user.id} via unified service")
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)